### PR TITLE
Adapt scripts for Windows builds

### PR DESCRIPTION
**Pull request by neomatrix369**
Thanks @sunnythepooh for testing this out on the windows VM and bringing it this far.

Changes made to make it work on the Windows machine:
- Amended a couple of functions to not pass additional configure params for Windows builds
- Added time measurement checks to see how much time cloning and updating Git repos took

### CODE REVIEW

**Comment by neomatrix369** [comment]
Kicked off windows build - https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk_build_x86-64_windows_mani_test/8/console which finished successfully

File: makejdk.sh [hunk 1]
```diff
@@ -306,6 +306,13 @@ checkInCaseOfDockerShouldTheContainerBePreserved
...
-cloneOpenJDKGitRepo
-getOpenJDKUpdateVersion
+time (
+    echo "Cloning OpenJDK git repo"
+    cloneOpenJDKGitRepo
+)
+
+time (
+    echo "Updating the OpenJDK git repo on local environment"
+    getOpenJDKUpdateVersion
```
Comment (karianna) [inline makejdk.sh]: Echo statement is incorrect?  Not sure why you're wrapping a time around this either?

Comment (neomatrix369) [inline makejdk.sh]: Aah this is incorrect, will remove that.

Comment (neomatrix369) [inline makejdk.sh]: Fixed via 		a2b0ca8

**Pull request merged**